src/pages/404.tsx

```diff
@@ -1,20 +1,30 @@
 import { useEffect, useState } from 'react'
 import type { NextPage } from 'next'
 import { useRouter } from 'next/router'
+import { AppRoutes } from '@/config/routes'
 
 // Rewrite the URL to put the Safe address into the query.
-const getRedirectUrl = (): string | undefined => {
-  if (typeof location === 'undefined') return
-
+export const _getRedirectUrl = (location: Location): string | undefined => {
   const { pathname, search } = location
   const re = /^\/([^/]+?:0x[0-9a-f]{40})/i
```

Comment:
Nit: We have a simliar expression in our url utils file called `prefixedAddressRe`. I would suggest moving this there or at least giving it a more descriptive name. I assume it also describes prefixed addresses but less restrictive when it comes to the shortname prefix?

Reply:
I wouldn't touch this at this time, so as not to break anything that isn't related to the tx id accidentally.

Reply:
A good mini-refactoring idea tho, feel free to make a tech debt issue.